net/tailscale/Makefile

```diff
@@ -77,6 +77,7 @@ endef
 
 define Package/tailscaled/conffiles
 /etc/config/tailscale
+/etc/tailscale
```

Comment:
You want to preserve state file, which is /etc/tailscale/tailscaled.state based on https://github.com/openwrt/packages/blob/db71747697934f5da65c458926fa5def60e19411/net/tailscale/files/tailscale.conf#L5 right? Thus I am not sure, why you want to preserve only /etc/tailscale

Reply:
```suggestion
/etc/tailscale/tailscaled.state
```